src/helpers/parameters/checkDuplication.js

```diff
@@ -4,13 +4,24 @@
  * @returns {array} an array of errors or an empty array if no error
  */
 export default (query) => {
-  const { limit, offset } = query;
+  const {
+    limit, offset, keyword, author, tag
+  } = query;
   const errors = {};
   if (Array.isArray(limit)) {
     errors.limit = 'Limit can not be declared more than once';
   }
   if (Array.isArray(offset)) {
     errors.offset = 'Offset can not be declared more than once';
   }
+  if (Array.isArray(keyword)) {
```

Comment:
I believe all these checks can be placed in one or statement and return a message that encapsulates all instances